src/backend/distributed/utils/resource_lock.c

```diff
@@ -1131,10 +1135,21 @@ AcquireDistributedLockOnRelations(List *relationIdList, LOCKMODE lockMode)
 			/* finish the partial lock statement */
 			if (startedLockCommand)
 			{
-				appendStringInfo(lockRelationsCommand, " IN %s MODE;\n", lockModeText);
+				appendStringInfo(lockRelationsCommand, lockTerminationString,
+								 lockModeText);
 				startedLockCommand = false;
 			}
 
+			/*
+			 * the user should not be able to trigger this codepath
+			 * with nowait = true or lockDescendants = flase since the
```

Comment:
typo: flase

Also, we generally start the multi-line comments with Capital letters